src/enums/contract-type.enum.ts

```diff
@@ -1,22 +1,22 @@
 export enum ContactType {
-   uop = 'UOP',
-   b2b = 'B2B',
-   uz_uod = 'UZ_UOD',
-   any = 'ANY',
+   UOP = 'uop',
+   B2B = 'b2b',
+   UZ_UOD = 'uz_uod',
+   ANY = 'any',
 }
 
 export const validateContractType = (contactType: string): string => {
-   if (contactType.toUpperCase() === ContactType.uop) {
-      return 'UOP';
+   if (contactType.toLowerCase() === ContactType.UOP) {
```

Comment:
Po prostu zróbcie sobie stringowe enumy. Będzie można zrobić `ContactType[contactType]`. Chyba nawet tak macie. Więc nie ma sensu robić tego tak na sztywno.

Reply:
Dzięki Kuba, wczoraj zdałem sobie sprawę, że nie za bardzo to coś daje, ale z drugiej strony zapomniałem to usunąć 😉